cupsfilters/texttopdf.c

```diff
@@ -2338,7 +2337,7 @@ write_line(int     row,			// I - Row number (0 to N)
 	  line ++;
 	}
 
-        for (i = 1; start < line; i ++, start ++)
+        for (; start < line; start ++)
```

Comment:
`i` is not used for anything, not even to determine when to end the loop. Remove it.